### DISCUSSION TITLE
Assets Management – Images/SVG

### DISCUSSION THREAD

**Discussion by santoshshinde2012** [NeoSOFT-Technologies/frontend-reactjs#15] [Ideas]
Assets Management – Images/SVG

**Comment by santoshshinde2012** [comment]
https://www.freecodecamp.org/news/how-to-structure-your-project-and-manage-static-resources-in-react-native-6f4cfc947d92/

**Comment by shinde87sagar** [comment]
Can the below points be more optimised approach? It can work for any frontend. It is more of a concept than tech related. 

Converting images to SVG and then the SVGs to font. 

- Step 1 advantages (PNG/JPG to SVG) :- PNG and JPG are size specific and scaling them for higher viewports will lead to pixelation. Scaling them to lower viewports will lead to higher sized image being compressed. Bandwidth issues. No need to keep multiple versions of the images for different view ports. Vector Graphics. 👍 

- Step 2 advantages (SVG to SVG custom font) :- Reduce number of server requests as individual fonts are compressed in 1 font file. New logos which are not used on the current page are also eagerly loaded due to font file. Scalable easily. Currently gives support for multi-color svgs as well. Bandwidth used is very very low. 

Reference
https://www.codingforentrepreneurs.com/blog/svg-icons-to-css-webfonts-to-cdn
https://css-tricks.com/using-custom-fonts-with-svg-in-an-image-tag/
